flycheck.el

```diff
@@ -1287,7 +1287,7 @@ If the version number could not be determined, signal an error,
 if called interactively, or if SHOW-VERSION is non-nil, otherwise
 just return nil."
   (interactive (list t))
-  (let ((version (pkg-info-version-info 'flycheck)))
+  (let ((version (package-get-version)))
```

Comment:
I think we should make this conditional, so we don't have to require Emacs 27 for this trivial function. E.g. we can just add the version as a variable and fall back to it in the absence of this function.